src/interface/src/app/treatments/map-stands-tx-result/map-stands-tx-result.component.scss

```diff
@@ -0,0 +1,86 @@
+@import "colors";
+@import "mixins";
+
+::ng-deep {
+  .treatment-tooltip {
+    @include public-sans();
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 22px;
+
+    .maplibregl-popup-content {
+      background-color: $color-black;
+      color: $color-white;
+      padding: 8px 12px;
+      border-radius: 8px;
+    }
+
+    // overriding tip colors for each corner
+
+    &.maplibregl-popup-anchor-right .maplibregl-popup-tip {
```

Comment:
open to suggestions on how to make all these overrides less verbose

Reply:
I don't know if it's possible to centralize these styles somewhere in the future, but I imagine we will probably reuse them.